gnucash/gnome-utils/gnc-tree-view.h

```diff
@@ -287,40 +287,24 @@ gnc_tree_view_add_pix_column (GncTreeView *view,
                               gint model_visibility_column,
                               GtkTreeIterCompareFunc column_sort_fn);
 
-/** This function adds a new combobox column to a GncTreeView base
- *  view.  The parameters it takes in common with
- *  gnc_tree_view_add_text_column() behave the same as there.  In
- *  addition, it will use combo_tree_model as the GtkTreeModel for the
- *  combobox, and combo_model_text_column will be the column in the
- *  model used for displaying the text in the combobox.
- */
-GtkTreeViewColumn *
-gnc_tree_view_add_combo_column (GncTreeView *view,
-                                const gchar *column_title,
-                                const gchar *pref_name,
-                                const gchar *sizing_text,
-                                gint model_data_column,
-                                gint model_visibility_column,
-                                GtkTreeModel *combo_tree_model,
-                                gint combo_model_text_column,
-                                GtkTreeIterCompareFunc column_sort_fn);
-
-
-/** This function adds a new date column to a GncTreeView base
- *  view.  The parameters it takes in common with
- *  gnc_tree_view_add_text_column() behave the same as there.
- */
-GtkTreeViewColumn *
-gnc_tree_view_add_date_column (GncTreeView *view,
-                               const gchar *column_title,
-                               const gchar *pref_name,
-                               const gchar *icon_name,
-                               const gchar *sizing_text,
-                               gint model_data_column,
-                               gint model_visibility_column,
-                               GtkTreeIterCompareFunc column_sort_fn);
-
-
+(??)/** This function adds a new combobox column to a GncTreeView base
+(??) *  view.  The parameters it takes in common with
+(??) *  gnc_tree_view_add_text_column() behave the same as there.  In
+(??) *  addition, it will use combo_tree_model as the GtkTreeModel for the
+(??) *  combobox, and combo_model_text_column will be the column in the
+(??) *  model used for displaying the text in the combobox.
+(??) */
+(??)GtkTreeViewColumn *
+(??)gnc_tree_view_add_combo_column (GncTreeView *view,
+(??)                                const gchar *column_title,
+(??)                                const gchar *pref_name,
+(??)                                const gchar *sizing_text,
+(??)                                gint model_data_column,
+(??)                                gint model_visibility_column,
+(??)                                GtkTreeModel *combo_tree_model,
+(??)                                gint combo_model_text_column,
+(??)                                GtkTreeIterCompareFunc column_sort_fn);
+(??)
```

Comment:
Looks like you kept the wrong decl here. I realize that you delete it again along with the definition of `gnc_tree_view_add_date_column` but having a build-breaker like this makes bisects difficult.

Reply:
Yep. Bad merge.